File: 202320_S2_E4_PremiosPeliculas_Back/src/main/java/co/edu/uniandes/dse/premiospeliculas/dto/PersonaDTO.java
package co.edu.uniandes.dse.premiospeliculas.dto;

import java.util.Date;


import lombok.Data;

@Data
public class PersonaDTO {
    private Long id;
    private String nombre;
    private String nacionalidad;
    private Date fechaNacimiento;
    private String biografia;
    private String rol;
}

File: 202320_S2_E4_PremiosPeliculas_Front/src/app/persona/personaDTO.ts
export class PersonaDTO {
    id: number;
    nombre: string;
    nacionalidad: string;
    fechaNacimiento: Date;
    biografia: string;
    rol: string;
    imagen:string;

    constructor(
        id: number,
        nombre: string,
        nacionalidad: string,
        fechaNacimiento: Date,
        biografia: string,
        rol: string,
        imagen:string
        
    ) {
        this.id = id;
        this.nombre = nombre;
        this.nacionalidad = nacionalidad;
        this.fechaNacimiento = fechaNacimiento;
        this.biografia = biografia;
        this.rol = rol;
        this.imagen = imagen;
    }
}
